### PR TITLE
claws-mail 4.1.0

### PR DESCRIPTION
**Pull request by Bo98**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/claws-mail.rb [hunk 1]
```diff
@@ -23,13 +23,19 @@ class ClawsMail < Formula
...
+  # Fix compile on non-X11 platforms (macOS).
+  # https://www.thewildbeast.co.uk/claws-mail/bugzilla/show_bug.cgi?id=4505
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/a9cc349d382629ff1e5c805b6f699f0c4a65a0e3/claws-mail/4.0.0-macos.patch"
+    sha256 "36f37cf26343b2b5fe51b6a8b3aa28c5824ecd655109d9e8f6cd88d24b2d351f"
+  end
+
   def install
     system "./configure", "--prefix=#{prefix}",
-                          "LDFLAGS=-Wl,-framework -Wl,Security",
```
Comment (Bo98) [inline Formula/claws-mail.rb:32]: I know this is currently making things fail, but this line is indicative of a problem with our libetpan - it's actually using the static library here which isn't ideal.

I'll fix it in the libetpan side.

Comment (Bo98) [reply Formula/claws-mail.rb:32]: #96869 will hopefully fix it.

File: Formula/claws-mail.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class ClawsMail < Formula
   desc "User-friendly, lightweight, and fast email client"
   homepage "https://www.claws-mail.org/"
-  url "https://www.claws-mail.org/releases/claws-mail-3.18.0.tar.gz"
-  sha256 "f107deec1f0debfae77f7f0022aef713f85a5b1f7d0b1a0e98f6eebe1e3556f2"
+  url "https://www.claws-mail.org/releases/claws-mail-4.0.0.tar.gz"
+  sha256 "6f04364715f97acd8c26c7edd61a57598d926f57e796c03bb9d9080cfa3dc894"
```
Comment (bayandin) [inline Formula/claws-mail.rb:5]: claws-mail 4.1 has been released

```suggestion
  url "https://www.claws-mail.org/releases/claws-mail-4.1.0.tar.gz"
  sha256 "534603d60200f3be346830d2fafd90dbdf0b59b9b3b928bc0aa7f8c1c218ec31"
```

**Comment by Bo98** [comment]
A patch is still needed - but it needs the one I submitted upstream rather than the one I backported for 4.0.0. I'll push it to formula-patches shortly.

**Pull request closed without merging**